core/src/epicli/cli/epicli.py

```diff
@@ -273,6 +273,8 @@ def splitarg(arg):
                             help='Absolute path to directory with build artifacts.')
 
     #optional
+    optional.add_argument('-f', '--file', dest='file', type=str, required=False,
+                            help='File with updated configuration definitions to use for the components to be updated.')
```

Comment:
```suggestion
                            help='File with updated configuration definitions to use for the components to be upgraded.')
```
IMO `upgraded` looks better since we have `epicli upgrade` 😉